packages/react-components/src/components/CameraButton.tsx

```diff
@@ -139,6 +148,9 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
   const [waitForCamera, setWaitForCamera] = useState(false);
   const localeStrings = useLocale().strings.cameraButton;
   const strings = { ...localeStrings, ...props.strings };
+  const [announcerString, setAnnouncerString] = useState<string>(
+    props.checked ? strings.cameraActionTurnedOnAnnouncement : strings.cameraActionTurnedOffAnnouncement
```

Comment:
my feeling is this needs to be undefined on first render, otherwise what stops this being read out on the first render pass?
```ts
const [announcerString, setAnnouncerString] = useState<string | undefined>();
```